mz_bokeh_package/components/plot_settings.py

```diff
@@ -704,6 +708,25 @@ def _init_plot_settings_values(self):
             self._set_setting_property(setting_id, value)
             self._set_setting_widget_value(setting_id, value)
 
+    @staticmethod
+    def _set_color_for_renderer(renderer: GlyphRenderer, new_color: str):
+        """Set line color and fill color of a bokeh renderer for its default representation,
+        as well as for the selection and nonselection glyphs.
+
+        Args:
+            renderer: a renderer to apply the function.
+            new_color: HEX value of the target color
+        """
+
+        if not renderer.selection_glyph or not renderer.nonselection_glyph:
+            return
```

Comment:
Is that an intended behaviour?

That means that dashboards that haven't defined the selection/nonselection glyphs will just fail here or simply never change the color, right? If you move the if statement two lines down, you get back to the original behaviour?